meta-zarhus-distro/recipes-tools/zarhus-memtools/zarhus-memtools_1.0.0.bb

```diff
@@ -0,0 +1,32 @@
+SUMMARY = "Tools for measuring and analyzing memory usage of a process"
+HOMEPAGE = "https://docs.zarhus.com"
+
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=5a6917ac6c34ca9b935f702e32987df1"
+
+SRC_URI = "\
+    file://mem_usage \
+    file://mem_test \
+    file://plot_mem_usage.py \
+    file://LICENSE \
+    file://heap_analyze \
+    "
+
+S = "${WORKDIR}"
+
+do_install() {
+    install -d ${D}${bindir}
+    install -m 0755 ${S}/mem_test ${D}${bindir}
+    install -m 0755 ${S}/plot_mem_usage.py ${D}${bindir}
+    install -m 0755 ${S}/mem_usage ${D}${bindir}
+    install -m 0755 ${S}/heap_analyze ${D}${bindir}
+}
+
+FILES:${PN} += " \
+    ${bindir}/mem_usage \
+    ${bindir}/mem_test \
+    ${bindir}/plot_mem_usage.py \
+    ${bindir}/heap_analyze \
+    "
+
+RDEPENDS:${PN} += "bash"
```

Comment:
I'm not particularly a fan of having X debug tools in the repository. I'd suggest creating a separate layer like some `meta-zarhus-debug` so they're easier to track. In the end we can merge it this way but let's then file some issue for discussion of architecture on zarhus testing/debuging.

Reply:
@macpijan what do you think?

Reply:
I personally think we should merge this for now and discuss a separate layer afterwards.

Reply:
> what do you think?

@PLangowski As discussed in the meeting, please create an issue, link here, and resolve  this thread for now.

Reply:
Done https://github.com/zarhus/zarhus-issues/issues/32
@tym2k1 please resolve this issue